rrfs-test/validated_yamls/templates/obtype_config/adpsfc_airTemperature_187.yaml

```diff
@@ -107,24 +104,32 @@
                  xvals: [110000, 105000, 100000, 95000, 90000, 85000, 80000, 75000, 70000, 65000, 60000, 55000, 50000, 45000, 40000, 35000, 30000, 25000, 20000, 15000, 10000, 7500, 5000, 4000, 3000, 2000, 1000, 500, 400, 300, 200, 100, 0]
                  errors: [2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585, 2.2585]
 
-         # Error inflation based on pressure check (setupt.f90)
+         # Reduce observation error and turn off ObsErrorFactorPressureCheck (l_gsd_terrain_match_surfTobs=.true., l_sfcobserror_ramp_t=.true.,)
          - filter: Perform Action
            filter variables:
            - name: airTemperature
-           where:
-           - variable: ObsType/airTemperature
-             is_in: 187
            action:
              name: inflate error
-             inflation variable:
-               name: ObsFunction/ObsErrorFactorPressureCheck
-               options:
-                 variable: airTemperature
-                 inflation factor: 8.0
-                 # The new feature "surface observation error ramp"
-                 # needs to be added to UFO to align with the ramp
-                 # options for temperature and humidity in GSI.
-                 #surface observation error ramp: true
+             inflation factor: 0.5
```

Comment:
Similar to the above, Line 107-113 is expected to be turned off (commented out) by default.